content/3.Joining Corda Network/4.Onboarding Workflow.md

```diff
@@ -1,29 +1,35 @@
 # Onboarding Workflow
 
-A summary of steps to join Corda Network (including the Pre-Production or Production network) is listed below. Before you proceed with these steps, please ensure that you have read and understood the following documentations before proceeding with these steps:
+Here's a summary of steps to join Corda Network. Before you proceed with the steps in this page, make sure that you have read and understood the following documentations:
 
 1. [Pre-requisite to join](https://corda.network/joining-corda-network/pre-requisites)
 2. [Corda Network Rulebook](https://corda.network/corda-network-rulebook/introduction) 
 3. [Corda Network SLA Handbook](https://corda.network/trust-centre/sla-&-maintenance-handbook)
 
 ## Step 1: Introductory Call and Terms of Use Agreement
 
-If you are decide to join Corda Network, you should send an email to info@corda.network. You should answer the following questions:
+If you are interested to join Corda Network, please send an email to info@corda.network with answers to these questions:
```

Comment:
If we are pushing the Contact Us form onto the website, we can remove the ask to email info@corda.network

Reply:
Yes good point - we can edit this point once the contact us form is available on the production website

Reply:
I think this whole step would change quite a lot once that form is ready

Reply:
Agree - this should be in place very shortly